### PR TITLE
Remove the warning messages for MSGraph migration

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
Remove the warning messages for MSGraph migration from Az.KeyVaut, Az.Resources, Az.Sql, Az.Aks

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Aks/Aks/ChangeLog.md [hunk 1]
```diff
@@ -18,8 +18,12 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+<<<<<<< HEAD
```
Comment (isra-fel) [inline src/Aks/Aks/ChangeLog.md:21]: Bad merge in this file

**Review by isra-fel — COMMENTED** [review]
Otherwise it LGTM

**Comment by BethanyZhou** [comment]
Resolved. Did I miss any other modules in your mind? @isra-fel I just did a text search.

**Review by isra-fel — APPROVED** [review]
StorageSync and Synapse also migrated to MSGraph. But they did add the warning message, so there's nothing more to do.

**Pull request merged**